bridge/client/app/_models/trace.ts

```diff
@@ -321,6 +321,10 @@ class Trace {
     return this.type.includes(".finished") ? this : this.traces.find(t => t.type.includes(".finished"));
   }
 
+  getDeploymentUrl() {
+    return this.data.deployment.deploymentURIsPublic[0];
```

Comment:
Why is `deploymentURIsPublic` an Array and is there a use case where multiple URIs are provided?

Reply:
@agrimmer ?